Add Documentation for custom hotkeys (#57)
Chatterino/wiki: docs/Hotkeys.md

## Introduction to Custom Hotkeys

By default Chatterino contains a set of default *hotkeys*. A hotkey can be described using five parameters:

> - Its name,

Reviewer: ```suggestion
- Its name
```

- A [category](#categories)
- An [action](#actions)
- A key combo
- The [arguments](#arguments)

![Open quick switcher hotkey being edited](images/hotkeys/Editing.png)

### Categories

> Categories describe where in the app the hotkey's action take place. For example:

Reviewer 2: ```suggestion
Categories describe where in the app the hotkeys action takes place. For example:
```
It stuck out since it was the only `hotkey's` vs all the other `hotkeys`

- Sending a message by pressing enter takes place in the **Split input box**,
- Closing the current split takes place in the current **Split**.

### Actions

An action simply describes what a hotkey does. For example:

- The **reconnect** action in a Split causes Chatterino to reconnect to Twitch/IRC.

> - the **zoom** action in a Window causes Chatterino to zoom in or out (depending on the [arguments][#hotkey-arguments] given)

Reviewer: ```suggestion
- The **zoom** action in a Window causes Chatterino to zoom in or out (depending on the [arguments][#hotkey-arguments] given)
```

> - The **Zoom in/out** action in a Window causes Chatterino to zoom in or out (depending on the [arguments][#hotkey-arguments] given)

Reviewer 2: ```suggestion
- The **Zoom in/out** action in a Window causes Chatterino to zoom in or out (depending on the [arguments](#hotkey-arguments) given)
```

> ## Introduction to custom hotkeys

Reviewer 3: ```suggestion
## Introduction to Custom Hotkeys
```

By default Chatterino contains a set of default *hotkeys*. A hotkey can be described using five parameters:

- Its name,

> - a [scope][scope],

Reviewer 3: ```suggestion
- A [scope](#scopes)
```

> - an [action][action],

Reviewer 3: ```suggestion
- An [action](#actions)
```

> - a key combo,

Reviewer 3: ```suggestion
- A key combo
```

> - and [arguments][arguments]

Reviewer 3: ```suggestion
- The [arguments](#arguments)
```

![Open quick switcher hotkey being edited](images/hotkeys/Editing.png)

> ### What is a scope

Reviewer 3: ```suggestion
### Scopes
```

Scopes describe where in the app the hotkey's action take place. For example:

> - sending a message by pressing enter takes place in the **Split input box**,

Reviewer 3: ```suggestion
- Sending a message by pressing enter takes place in the **Split input box**,
```

> - closing the current split takes place in the current **Split**.

Reviewer 3: ```suggestion
- Closing the current split takes place in the current **Split**.
```

> ### What is an action

Reviewer 3: ```suggestion
### Actions
```

An action simply describes what a hotkey does. For example:

> - the **reconnect** action in a Split causes Chatterino to reconnect to Twitch/IRC.

Reviewer 3: ```suggestion
- The **reconnect** action in a Split causes Chatterino to reconnect to Twitch/IRC.
```

> - the **zoom** action in a Window causes Chatterino to zoom in or out (depending on the [arguments][arguments] given)

Reviewer 3: ```suggestion
- the **zoom** action in a Window causes Chatterino to zoom in or out (depending on the [arguments][#hotkey-arguments] given)
```

> - the **zoom** action in a Window causes Chatterino to zoom in or out (depending on the [arguments][arguments] given)

Reviewer 3: ```suggestion
- The **zoom** action in a Window causes Chatterino to zoom in or out (depending on the [arguments][arguments] given)
```

> - The **Zoom in/out** action in a Window causes Chatterino to zoom in or out (depending on the [arguments](#hotkey-arguments) given)

Reviewer 2: ```suggestion
- The **Zoom in/out** action in a Window causes Chatterino to zoom in or out (depending on the [arguments](#arguments) given)
```
missed this the first time

> ### What are hotkey arguments

Reviewer 3: ```suggestion
### Hotkey Arguments
```

Hotkey arguments describe the specifics of hotkeys. For example:

> - the **zoom** action in a Window takes a single argument (`in` or `out`) which tells Chatterino to either zoom in or zoom out.

Reviewer 3: ```suggestion
- The **zoom** action in a Window takes a single argument (`in` or `out`) which tells Chatterino to either zoom in or zoom out.
```

> 
> [arguments]:#what-are-hotkey-arguments
> [action]:#what-is-an-action
> [scope]:#what-is-a-scope

Reviewer 3: ```suggestion
```
